ui/ui-components/pages/settings/[tab].tsx

```diff
@@ -7,13 +7,14 @@ import { useRouter } from "next/router";
 import { Models, Actions } from "../../utils/apiData";
 import SettingCard from "../../components/settings/SettingCard";
 import ImportAndUpdateAllRecords from "../../components/settings/ImportAndUpdate";
-import IdentifyingProperty from "../../components/settings/IdentifyingProperty";
 import ResetCluster from "../../components/settings/ResetCluster";
 import ResetData from "../../components/settings/ResetData";
 import ResetCache from "../../components/settings/ResetCache";
 import { ErrorHandler } from "../../utils/errorHandler";
 import { SuccessHandler } from "../../utils/successHandler";
 
+const settingsWhichTriggerInterfaceReload = [["core", "cluster-name"]];
```

Comment:
I typically wrap these in an `Object.freeze()` so that they're read only and use `ALL_CAPS` to identify as a constant.

Reply:
I saw that! Can you share more about the benefit?  Does it help with memory usage or something like that?